--- a/SPECS/fwupd.spec
+++ b/SPECS/fwupd.spec
@@ -31,11 +31,7 @@
 %global have_uefi 1
 %define sb_ca		%{_datadir}/pki/sb-certs/secureboot-ca-%{_arch}.cer
 %define sb_cer		%{_datadir}/pki/sb-certs/secureboot-fwupd-%{_arch}.cer
-%if 0%{?centos}
-%define sb_key		centossecureboot202
-%else
-%define sb_key		redhatsecureboot802
-%endif
+%define sb_key		openelasecureboot
 %endif
 
 # flashrom is only available on these arches
